packages/data/addon/adapters/facts/elide.ts

```diff
@@ -156,9 +193,31 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
    * @param _request
    * @param _options
    */
-  async urlForDownloadQuery(_request: RequestV1, _options: RequestOptions): Promise<string> {
-    return 'TODO';
+  async urlForDownloadQuery(request: RequestV1, options: RequestOptions = {}): Promise<string> {
+    let response = await this.fetchDataForExportTask.perform(request, options);
+    let url = response.tableExport.edges[0].node.result?.url || '';
```

Comment:
If there is an error in the request, I believe `edges` could be an empty array so we should account for that here.